### PR TITLE
enable 3.11 release for grafana-operator

### PR DESCRIPTION
**Pull request by Tirumalavasa**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Specify your PR reviewers**:

/cc @gyliu513 @morvencao

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: prow/cluster/jobs/IBM/ibm-monitoring-grafana-operator/IBM.ibm-monitoring-grafana-operator.release-1.11.yaml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        name: ""
+        securityContext:
+          privileged: true
+    trigger: '(?m)^/test (?:.*? )?test(?: .*?)?$'
+
+postsubmits:
+  IBM/ibm-monitoring-grafana-operator:
+  - name: check_ibm-monitoring-grafana-operator_postsubmit
+    branches:
+    - ^master$
```
Comment (horis233) [inline prow/cluster/jobs/IBM/ibm-monitoring-grafana-operator/IBM.ibm-monitoring-grafana-operator.release-1.11.yaml:61]: @Tirumalavasa Please update all `^master$` to `^release-1.11$`

Comment (Tirumalavasa) [reply prow/cluster/jobs/IBM/ibm-monitoring-grafana-operator/IBM.ibm-monitoring-grafana-operator.release-1.11.yaml]: @horis233 , Thank you for pointing this. Fixed it. Please check now.

**Review by horis233 — APPROVED** [review]
/lgtm

**Pull request merged**